packages/schema/src/workspace-invitations/index.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from 'zod'
+import { PageRequestSchema, PageResponseSchema } from '@/pagination'
+
+export const InviteMemberResponseSchema = z.object({
```

Comment:
The return type contains some mismatched keys. Refer [this](https://github.com/keyshade-xyz/keyshade/pull/586/files#diff-a92613f23cf2834d52eaedb87c4e8c460d48b6fca8df4ccc968b2a8af19a9c3fR432-R450) to understand the exact type. Alternatively, you can also execute the API query to get the response type.